pkg/rtc/participant_signal.go

```diff
@@ -265,7 +267,11 @@ func (p *ParticipantImpl) writeMessage(msg *livekit.SignalResponse) error {
 	}
 
 	err := sink.WriteMessage(msg)
-	if err != nil {
+	if errors.Is(err, psrpc.Canceled) {
+		p.params.Logger.Debugw("could not send message to participant",
+			"error", err, "messageType", fmt.Sprintf("%T", msg.Message))
+		return nil
```

Comment:
Not related to this PR, this function should probably be moved to participant_signal.go